content/sensu-core/1.5/installation/installing-plugins.md

```diff
@@ -46,19 +49,30 @@ Successfully installed sensu-plugins-disk-checks-3.1.0
 1 gem installed
 [SENSU-INSTALL] successfully installed Sensu plugins: ["sensu-plugins-disk-checks"]{{< /highlight >}}
 
+### Pinning Versions
+
+We strongly recommend pinning the versions of any plugins installed in production.
+To install the [Sensu Disk Checks Plugin][3] and pin the version, use:
+
+{{< highlight shell >}}
+sudo sensu-install -p 'sensu-plugins-disk-checks:3.1.0'
+{{< /highlight >}}
+
```

Comment:
Since this page doesn't mention configuration management, I'm going to hold off linking to this page for now. #712 is tracking more general efforts around incorporating these awesome best practices into the docs.

Reply:
Not sure why that matters (config management), seems like duplicate effort but OK.

Reply:
Updated to link to the best practices doc!